app/views/Moderation/Schools/index.tsx

```diff
@@ -203,6 +229,61 @@ function SchoolItem(props: SchoolItemProps) {
         },
     );
 
+    const [
+        updateUserActiveStatus,
+        { loading: userActiveLoading },
+    ] = useMutation<
+        UpdateUserActiveStatusMutation,
+        UpdateUserActiveStatusMutationVariables
+    >(
+        UPDATE_USER_ACTIVE_STATUS,
+        {
+            onCompleted: (response) => {
+                const userDeactivateToggle = response?.moderatorMutation?.userDeactivateToggle;
+                if (!userDeactivateToggle) {
+                    return;
+                }
+
+                const {
+                    errors,
+                    ok,
+                } = userDeactivateToggle;
+
+                if (ok) {
+                    onUserDeactivationSuccess();
+                    alert.show(
+                        'User deactived successful',
+                        { variant: 'success' },
+                    );
+                } else if (errors) {
+                    const transformedError = transformToFormError(
+                        removeNull(errors) as ObjectError[],
+                    );
+                    alert.show(
+                        <ErrorMessage
+                            header="Account Deactivated Successfully"
+                            description={
+                                isDefined(transformedError)
+                                    ? transformedError[internal]
+                                    : undefined
+                            }
+                        />,
```

Comment:
if errors this error message heading doesn't make sense. Please update it accordingly.